src/drivers/fluid_winmidi.c

```diff
@@ -28,35 +28,60 @@
  * 1.1)For example, if the user chooses 2 devices at index 0 and 1, the user
  * must specify this by putting the name "0;1" in midi.winmidi.device setting.
- * We get a fictif device composed of real devices (0,1). This fictif device
+ * We get a fictive device composed of real devices (0,1). This fictive device
  * behaves like a device with 32 MIDI channels whose messages are forwarded to
  * driver output as this:
  * - MIDI messages from real device 0 are output to MIDI channels set 0 to 15.
- * - MIDI messages from real device 1 are output to MIDI channels set 15 to 31.
+ * - MIDI messages from real device 1 are output to MIDI channels set 16 to 31.
  *
  * 1.2)Now another example with the name "1;0". The driver will forward
  * MIDI messages as this:
  * - MIDI messages from real device 1 are output to MIDI channels set 0 to 15.
- * - MIDI messages from real device 0 are output to MIDI channels set 15 to 31.
+ * - MIDI messages from real device 0 are output to MIDI channels set 16 to 31.
  * So, the device order specified in the setting allows the user to choose the
  * MIDI channel set associated with this real device at the driver output
  * according this formula: output_channel = input_channel + device_order * 16.
  *
+ * Note the driver ignores the setting synth.midi-channels which default value
+ * for this setting is 16, in which case the second MIDI device will not be hear
+ * on the fluid synth instance.
+ * a) You have to rise the setting synth.midi-channels to accept the number
+ * of MIDI channels provided by the MIDI driver.
+ * b) You can also keep the default value of 16 for synth.midi-channels
+ * and have your application connected between the MIDI driver and the fluid synth.
+ * This way your application is able to map any MIDI channel from the driver to
+ * any synths that could be a fluid synth and hardware synth(s).
+ *
  * 2)Note also that the driver handles single device by putting the device name
  * in midi.winmidi.device setting.
  * The user can set the device name "0:Port MIDI SB Live! [CE00]" in the setting.
  * or use the multi device naming "0" (specifying only device index 0).
  * Both naming choice allows the driver to handle the same single device.
  *
+ * 3)If the setting "midi.autoconnect" is enabled and the device name is "default",
+ * then all the available devices are opened, applying the appropriate channel
+ * mappings to each device (the first device is mapped to the 16 first channels
+ * (0-15), the second one to the next 16 channels (16-31), and so on with the limit
+ * of the synth.midi-channels setting. After arriving to the channels limit, the
+ * mapping restarts with the channels 0-15.
+ * This makes all MIDI devices audible on the synth instance. However with more than
+ * 1 MIDI devices and the default value of synth.midi-channels of 16 then MIDI
+ * channels conflicts are possible if MIDI devices transmit on the same MIDI channel.
```

Comment:
Think of this scenario: The user connects two MIDI controllers to the MIDI IN ports of a [MIDI merge box](https://www.thomann.de/intl/midi_solutions_merger_v2.htm), and the MIDI OUT of the merge box to the input of Fluidsynth (or any other synthesizer). Then the user plays both controllers in the same MIDI channel. This may produce event collisions, of course, because that is how MIDI works. Each MIDI controller should use a different MIDI channel.